components/block_collection/stylesheets/block_collection.styl

```diff
@@ -1,16 +1,5 @@
-/* GENERAL */
-borderColor(color)
-  rgba(color, 30%)
```

Comment:
I removed this guy for a few reasons: it's weird that we have to use snakecase to work around a naming conflict, it's one line so it's hardly worth wrapping, and I think the thing we're really abstracting is just the 30%.